src/config/env/production.js

```diff
@@ -0,0 +1,15 @@
+export default {
+    env: 'production',
+    db: 'mongodb://localhost/db',
+    port: process.env.SH_BACKEND_PORT || 3000,
+    host: '0.0.0.0',
+    seedDB: true,
+    staticPath: process.env.SH_PATH_FRONTENT_DIST,
+    plugAndPlay: false,
+    mqtt: {
```

Comment:
I will very appreciate if you change mqtt parameters to pass it via env (as in heroku.js) from parent npm modulle (smart-house)